<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnippetWidget</class>
 <widget class="QWidget" name="SnippetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="ui_cbSnipptBook">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_pbEdit">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="ui_lwSnippets">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_cbSnipptBook</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>SnippetWidget</receiver>
   <slot>loadSnippets(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_lwSnippets</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>SnippetWidget</receiver>
   <slot>insertSnippet(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>editSnippets()</signal>
  <slot>loadSnippets(QString)</slot>
  <slot>insertSnippet(QListWidgetItem*)</slot>
 </slots>
</ui>
